src/ui.rs

```diff
@@ -0,0 +1,109 @@
+use bevy::{prelude::*, utils::HashSet};
+use bevy_egui::{
+    egui::{self},
+    EguiContexts, EguiPlugin,
+};
+
+use crate::animation_loader::{AnimationSelectEvent, BvhFile};
+
+pub struct UiPlugin;
+
+impl Plugin for UiPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_systems(Update, ui)
+            .add_plugins(EguiPlugin)
+            .insert_resource(ShowDrawArrow { show: true })
+            .insert_resource(SelectedFiles::default())
+            .add_event::<DrawArrowEvent>();
```

Comment:
We don't need this `Event`.
```suggestion
            .insert_resource(SelectedFiles::default()));
```